### PR TITLE
Mark Client Packages Internal

### PR DESCRIPTION
**Pull request by anthony-murphy**
For our next major version Fluid Framework will be using explicit release tagging, and d.ts rollups to support public, beta, alpha, and internal. In preparation for that we are tagging everything internal to start, and then will carve out our other releases. Marking everything internal serves 2 purposes. 1) tags are inconsistently applied today, and it is easier to carve out a consistent set for each release type, rather than finding and fixing each miss application 2) which supports 1, inconsistent tags make tooling very hard and error prone to rollout, by making everything internal we have a consistent surface to enable our tooling upon, and this tooling will be used to inform our official carve outs.

related: 
https://github.com/microsoft/FluidFramework/pull/18409
https://github.com/microsoft/FluidFramework/pull/18408
https://github.com/microsoft/FluidFramework/pull/18411
https://github.com/microsoft/FluidFramework/pull/18412
https://github.com/microsoft/FluidFramework/pull/18410
https://github.com/microsoft/FluidFramework/pull/18413

### CODE REVIEW

**Comment by markfields** [comment]
@anthony-murphy What's the context here?

**Comment by alexvy86** [comment]
> @anthony-murphy Tony Murphy FTE What's the context here?

Just for the kicks 😝 .

Must be [this thread](https://teams.microsoft.com/l/message/19:50292e8934024fc19d6ca2080dd7681e@thread.skype/1700509218013?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=9ce27575-2f82-4689-abdb-bcff07e8063b&parentMessageId=1700509218013&teamName=Fluid%20Framework&channelName=Dev&createdTime=1700509218013) (msft internal). Josh told us about it in standup.

**Comment by anthony-murphy** [comment]
> > @anthony-murphy Tony Murphy FTE What's the context here?
> 
> Just for the kicks 😝 .
> 
> Must be [this thread](https://teams.microsoft.com/l/message/19:50292e8934024fc19d6ca2080dd7681e@thread.skype/1700509218013?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=9ce27575-2f82-4689-abdb-bcff07e8063b&parentMessageId=1700509218013&teamName=Fluid%20Framework&channelName=Dev&createdTime=1700509218013) (msft internal). Josh told us about it in standup.

i also improved the description. Ya'll just go here too quick :)

File: packages/dds/matrix/src/matrix.ts [hunk 1]
```diff
@@ -397,7 +396,7 @@ export class SharedMatrix<T = any>
 		}
 	}
 
-	/** @internal */ public _undoRemoveCols(colStart: number, spec: IJSONSegment) {
+	/***/ public _undoRemoveCols(colStart: number, spec: IJSONSegment) {
```
Comment (Josmithr) [inline packages/dds/matrix/src/matrix.ts:399]:
```suggestion
	public _undoRemoveCols(colStart: number, spec: IJSONSegment) {
```

File: packages/dds/matrix/src/matrix.ts [hunk 2]
```diff
@@ -374,7 +373,7 @@ export class SharedMatrix<T = any>
 		this.submitRowMessage(this.rows.remove(rowStart, count));
 	}
 
-	/** @internal */ public _undoRemoveRows(rowStart: number, spec: IJSONSegment) {
+	/***/ public _undoRemoveRows(rowStart: number, spec: IJSONSegment) {
```
Comment (Josmithr) [inline packages/dds/matrix/src/matrix.ts:376]:
```suggestion
	public _undoRemoveRows(rowStart: number, spec: IJSONSegment) {
```

File: packages/dds/merge-tree/src/attributionCollection.ts [hunk 3]
```diff
@@ -54,9 +54,7 @@ export interface IAttributionCollectionSpec<T> {
  * @sealed
  */
 export interface IAttributionCollectionSerializer {
-	/**
-	 * @internal
-	 */
+	/***/
```
Comment (Josmithr) [inline packages/dds/merge-tree/src/attributionCollection.ts:57]:
```suggestion
```

**Review by Josmithr — APPROVED** [review]
It looks like the tool you used for this left some empty `/****/` comments in places where there were previously comments only containing tags that were then removed. We should clean these up, though that could always be in a follow-up if we're worried about conflicts.

**Pull request merged**